tasks/modules.py

```diff
@@ -340,7 +340,13 @@ def generate_dummy_package(ctx, folder):
                         ctx.run(
                             "go mod edit -replace github.com/open-telemetry/opentelemetry-collector-contrib/connector/datadogconnector=github.com/open-telemetry/opentelemetry-collector-contrib/connector/datadogconnector@v0.103.0"
                         )
-
+                    if (
+                        mod.import_path == "github.com/DataDog/datadog-agent/comp/otelcol/configstore/impl"
+                        or mod.import_path == "github.com/DataDog/datadog-agent/comp/otelcol/configstore/def"
+                    ):
+                        ctx.run("go mod edit -exclude github.com/knadh/koanf/maps@v0.1.1")
+                        ctx.run("go mod edit -exclude github.com/knadh/koanf/providers/confmap@v0.1.0")
```

Comment:
why do we need this ?

Reply:
See: https://dd.slack.com/archives/C06PBHLD4DQ/p1721057665355079